<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelPassword">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpen">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ouvrir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDelete">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>250</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditUsername">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>301</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelUsername">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSuscribe">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>S'inscrire</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPassword">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUploadFolder">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNewFolder">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Créer Dossier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnect">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Se connecter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUploadFile">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDownload">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Télécharger</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRename">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Renomer</string>
    </property>
   </widget>
   <zorder>listWidget</zorder>
   <zorder>labelPassword</zorder>
   <zorder>pushButtonOpen</zorder>
   <zorder>pushButtonDelete</zorder>
   <zorder>lineEditUsername</zorder>
   <zorder>labelUsername</zorder>
   <zorder>pushButtonSuscribe</zorder>
   <zorder>lineEditPassword</zorder>
   <zorder>pushButtonUploadFolder</zorder>
   <zorder>pushButtonNewFolder</zorder>
   <zorder>pushButtonConnect</zorder>
   <zorder>pushButtonUploadFile</zorder>
   <zorder>pushButtonDownload</zorder>
   <zorder>pushButtonRename</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>485</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuOutil">
    <property name="title">
     <string>Outil</string>
    </property>
    <addaction name="actionModifier_Donnees"/>
    <addaction name="actionModifier_Password"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionA_Propos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuOutil"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionModifier_Donnees">
   <property name="text">
    <string>Modifier Donnees</string>
   </property>
  </action>
  <action name="actionModifier_Password">
   <property name="text">
    <string>Modifier Password</string>
   </property>
  </action>
  <action name="actionA_Propos">
   <property name="text">
    <string>A Propos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
